examples/vanilla-ts-esm/public/mux-player-cuepoints.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width" />
+    <title>&lt;mux-player&gt; example</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/water.css@2/out/water.css">
+    <link rel="stylesheet" href="./styles.css">
+    <script
+      defer
+      src="https://www.gstatic.com/cv/js/sender/v1/cast_sender.js?loadCastFramework=1"
+    ></script>
+    <script type="module" src="./dist/mux-player.js"></script>
+    <style>
+      mux-player {
+        display: block;
+        width: 100%;
+        margin: 1rem 0 2rem;
+        background-color: #000;
+        line-height: 0;
+      }
+
+      mux-player:not([audio]) {
+        aspect-ratio: 16 / 9;
+      }
+
+      #cuepoint-renderer {
+        background-color: lightgray;
+        padding: 10px;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <div class="left-header">
+        <a class="mux-logo" href="https://www.mux.com/player" target="_blank">
+          <img width="81" height="26" src="./images/mux-logo@2x.webp" alt="Mux logo" decoding="async">
+        </a>
+        <h1><a href="/">Elements</a></h1>
+      </div>
+      <div class="right-header">
+        <a class="github-logo" href="https://github.com/muxinc/elements" target="_blank">
+          <img width="32" height="32" src="./images/github-logo.svg" alt="Github logo">
+        </a>
+      </div>
+    </header>
+
+    <mux-player
+      stream-type="on-demand"
+      playback-id="23s11nz72DsoN657h4314PjKKjsF2JG33eBQQt6B95I"
+      id="cued-player"
+    ></mux-player>
+    <div>
+      <b>Active CuePoint:</b> <pre id="cuepoint-renderer">Unset</pre>
+    </div>
+    <script>
+      const playerEl = document.getElementById('cued-player');
+
+      function addCuePointsToPlayer() {
+        const cuePoints = [
+          { timestamp: 1, value: 'Simple Value' }, 
```

Comment:
nit (non-blocking): why not the shorter wording `time`, saves some characters

Reply:
@muxinc/open-elements Thoughts here? Eventually, we'll plausibly (somehow) support wall clock times (for live) and potentially ranges. If folks feel strongly one way or another, now would be the time to change this key name.

Reply:
refactored to `time` per out of band convo. 🎉